src/routes/add-place.ts

```diff
@@ -74,6 +75,9 @@ export default async function addPlace(fastify: FastifyInstance) {
           },
         });
       }
+
+      // back to places list
+      resp.header('HX-Redirect', `/`);
```

Comment:
this file `add-place.ts` should not be in this PR because you just made white space changes - please revert this commit.

Reply:
i see there's a change on this line - I'll resolve this conversation.